Resources/Prototypes/_CD/Entities/Objects/Misc/garmentbag.yml

```diff
@@ -17,3 +17,22 @@
       - id: PlushieLizard
     sound:
       path: /Audio/Effects/unwrap.ogg
+
+- type: entity
+  suffix: POINTMAN
```

Comment:
I don't think this should be in all caps, as that's usually reserved for "DON'T FUCKING MAP THIS" or "THIS IS A DEBUG OR ADMIN ONLY TOOL, DON'T TOUCH IT" But I'm not going to hold up the whole PR over it.

Reply:
force of habit, fixed now